### ISSUE TITLE
No broken line in the resulting picture

### ISSUE THREAD

**Issue by yyqxnice** [iffiX/PGPPortfolio-pytorch#2]
Thank you for your answer to my last question.When I am running the command line

python main.py  --mode plot --working_dir test --offline --algos nn,best,crp,rmr,ons

I met two problems.1)The command line will be stuck and there is no broken line in the resulting picture.
                                2)I want to know the range of training set and test set used by this code. The data set I downloaded through the command line is 162M. I don’t know if it is the same size as your original data set.
I hope to get your answer,thank you.

**Comment by iffiX** [comment]
For the stuck problem, I am not sure where the problem could occur as this code is run on our private cluster, and not fully tested across all platforms and environments. Sorry I cannot help you with that problem.

For the second problem, our configuration files are placed in [results](https://github.com/iffiX/PGPortfolio-pytorch/tree/master/results). As a reference, the sqlite database size is 236M when date range is `2015/05/01` to `2017/04/27`.

**Comment by yyqxnice** [comment]
Thank you for your answer to my question. I wonder if you can send me the 236M data set, because I ran it again and found that it is still 162M data.

**Comment by iffiX** [comment]
@yyqxnice Here is the link: https://drive.google.com/file/d/1ITKBNLyTWhtQDg8Lv3hQzM31NdWSL5kG/view?usp=sharing
